### DISCUSSION TITLE
How to configure paths to data

### DISCUSSION THREAD

**Discussion by Yorko** [executablebooks/meta#589] [Q&A] [answered]
Hi!

Sorry if it's a duplicate but I tried to search. How do I specify local paths to data files and other artifacts? E.g. how to read a file into a pandas DataFrame? For me `pd.read_csv('data.csv')` wouldn't work even if `data.csv` is placed in the same folder as the `.md` file. Also, if I print the current working directory `print(Path('.').absolute())` it shows smth like `/private/var/folders/xs/x4nxtnxd4qq06sg5z0x951q1q7mkqq/T/tmpi2kgtaoz`. Is that expected? Is there an option to configure paths so that I can work with local files?

**Comment by chrisjsewell** [comment, marked as answer]
See https://jupyterbook.org/content/execute.html#the-execution-working-directory.
Perhaps you are running an old version?

**Comment by Yorko** [comment]
Hi @chrisjsewell!

Thanks! Mine is the latest `0.12.1`, I'll run the steps to reproduce the issue, and will add details here.

**Comment by Yorko** [comment]
Hm, I created a blank jupyter book as explained [in the docs](https://jupyterbook.org/start/create.html) and it works, points to the local dir. Should've checked earlier, will be looking for a misconfiguration in my book. Thanks, @chrisjsewell

**Comment by Yorko** [comment]
Figured it out. That's what the flag `run_in_temp` in the `execution` section is controlling (as Chris suggested). If it's turned on, the current execution directory will look like `/private/var/folders/xs/x4nxtnxd4qq06sg5z0x951q1q7mkqq/T/tmpi2kgtaoz`, otherwise it points to the same directory where the notebook is placed.
